packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/MultiLabelEdge.tsx

```diff
@@ -38,31 +59,37 @@ export const MultiLabelEdge = memo(
       targetPosition,
     });
 
-    const { beginLabel, endLabel, centerLabel } = data || {};
+    const { beginLabel, endLabel, label, faded } = data || {};
 
     return (
       <>
-        <BaseEdge id={id} path={edgePath} style={style} markerEnd={markerEnd} markerStart={markerStart} />
+        <BaseEdge
+          id={id}
+          path={edgePath}
+          style={multiLabelEdgeStyle(theme, style, selected, faded)}
+          markerEnd={selected ? `${markerEnd?.slice(0, markerEnd.length - 1)}--selected)` : markerEnd}
+          markerStart={selected ? `${markerStart?.slice(0, markerStart.length - 1)}--selected)` : markerStart}
+        />
         <EdgeLabelRenderer>
           {beginLabel && (
             <Label
               transform={`translate(2%, 0%) translate(${sourceX}px,${sourceY}px)`}
               label={beginLabel}
-              faded={false}
+              faded={faded ? true : false}
```

Comment:
You could use `faded={faded || false} />`

Reply:
Fixed in https://github.com/eclipse-sirius/sirius-web/pull/2319 (This PR is based upon https://github.com/eclipse-sirius/sirius-web/pull/2319)